doc/source/conf.py

```diff
@@ -57,14 +57,23 @@
     "sphinxcontrib.apidoc",  # automatically generate API docs, see https://github.com/rtfd/readthedocs.org/issues/1139
     "sphinxcontrib.bibtex",
     "nbsphinx",
-    "nbsphinx_link",  # for linking notebooks from outside sphinx source root
     "myst_parser",
 ]
 
-# nbsphinx settings
+# -- nbsphinx settings -------------------------------------------------------
 nbsphinx_execute = "auto"
 
-# Bibliography
+# Create symlinks for example notebooks
+import glob
+nb_files = [os.path.basename(f) for f in glob.glob(os.path.join('examples','*.ipynb')) 
+        if not os.path.basename(f).startswith('temp_')]
+for nb_file in nb_files:
+    target = os.path.join('../../examples', nb_file)
+    if os.path.exists(target):
+        os.remove(target)
+    os.symlink(os.path.join('../doc/source/examples', nb_file), target)
```

Comment:
Can you explain how this is intended to work? Symlinks are created on every build docs run, but how is this related to the new `.ipynb` files under `examples` which point to the real files under `doc/source/examples`?

Reply:
So the bare `.ipynb` files in `examples/` are actually just symlinks pointing to the real ones under `doc/source/examples/`. This enables one to run `jupyter-notebook examples/<>.ipynb` and open the notebook `doc/source/examples/<>.ipynb`.

See below:
![image](https://user-images.githubusercontent.com/32061685/144891506-b41148e2-cad6-4aeb-b360-45f813adb286.png)

Reply:
Ah so those are the same links that were generated running `conf.py` and commited to the repo?

Reply:
yep! I thought it might be nice to create them via the `conf.py` to reduce the change of us forgetting to add symlinks when we create new notebooks?

Reply:
Sounds good to me, would need to add an instruction to commit the symlinks for new notebooks also.

Reply:
@jklaise I've added instructions in the docs readme in the alibi version of this PR: https://github.com/SeldonIO/alibi/pull/554.